### PR TITLE
chore: Relieve dependence on colors

### PR DESCRIPTION
**Pull request by kriskowal**
The compromised `colors` package is pervasive enough that both `http-server` and `node-static` depend upon it. I’m comfortable just saying informally that you need to run `python3 -m http.server` for the demo.

### CODE REVIEW

**Comment by warner** [comment]
Wanna change the `demo:` line to just use that python snippet, in lieu of documentation? Anyone reading that should be able to figure out how to run some equivalent command if they don't have python around.

**Comment by warner** [comment]
Oh wait should that be `python3 -m http.server demos` ? The special directory is probably not 100% obvious.

**Comment by kriskowal** [comment]
Posted an update with the Python demo script. The demos depend on serving the SES directory, to get to the built lockdown script, so `-d demos` is not necessary.

**Review by warner — APPROVED** [review]

File: packages/ses/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "build": "node scripts/bundle.js",
     "clean": "rm -rf dist",
     "cover": "c8 ava",
-    "demo": "http-server -o /demos",
+    "demo": "python3 -m http.server",
```
Comment (mhofman) [inline packages/ses/package.json:51]: This is unfortunate, we shouldn't force a python dependency just for a static server. There ought to be a simple enough node based static http server! Doesn't [serve](https://www.npmjs.com/package/serve) fit the bill?

**Pull request merged**